foyer/forcefields/xml/CHANGELOG.md

```diff
@@ -5,7 +5,10 @@
 v0.0.1 - April 15, 2021
  - started versioning of forcefield xmls
 v0.0.2 - June 24, 2021
- - update SMARTS string for opls_468
+ - update SMARTS string for opls_180 (from `[C;X4]([O;%opls_180])(H)(H)` to `[C;X4]([O;%opls_180])(H)(H)C`)
+ - update SMARTS string for opls_468 (from `[C;X4]([O;%opls_467])(H)(H)` to `[C;X4]([O;%opls_467])(H)(H)H`)
+ - update SMARTS string for opls_469 (from `H[C;%opls_468]` to `H[C;%opls_468,%opls_490]`)
+ - add SMARTS string for opls_490 (`[C;X4]([O;%opls_467])(H)(H)C`; ethoxy carbon
  - update references for opls_465 to opls_469
```

Comment:
One additional thing, let’s list these out as well. I think for most changes we make, we should list each affected atomtype as well. Thanks for already adding in the other changes I requested!

Reply:
by affected atomtype, do you mean those new overrides?

Reply:
Yes, but also opls 465 to 469 should all be listed, since each of them changed their citation.

Reply:
done